labzen/labzen.py

```diff
@@ -1,21 +1,75 @@
-import pandas as pd
+from nbformat import read, NO_CONVERT
+import os
+from pathlib import Path
+import re
+import glob
 
 
-def parse_lab(file_name: str) -> list:
+def parse_lab(notebook = 'no input'):
+    
     """Parse MDS lab files to return the markdown content
-
     Args:
         file_name (str): A path or list of paths to MDS lab files (either
             .ipynb or .Rmd). If left blank, the function will recursively
             search for all labs in the working directory based on the file
             extension.
     Returns:
         list: Each element of list is a content of one markdown cell.
-
     Example:
         parse_lab()
     """
-    return None
+    
+    # If the user did not defined the specific file, recursively 
+    # search for rmd and ipynb files in the working directory
+    if notebook == 'no input':
+        directory = os.getcwd()
+        types = ["*.ipynb", "*.Rmd"]
+        files = []
+        for type in types:
+            pathname = directory + "/**/*" + type
+            type_files = glob.glob(pathname, recursive=True)
+            files += type_files
+        names = [str(n+1) + '.' + os.path.basename(file)  for n, file in enumerate(files)]
```

Comment:
Some places we use `'` and other places we use `"`. Maybe we should stick to `"`? What do you think?

Reply:
Totally agree